src/helpers/device.js

```diff
@@ -0,0 +1,64 @@
+import { Dimensions } from 'react-native';
+
+// identify iPhone model based on device width
+const identifyDevice = deviceScreenProps => {
+  if (deviceScreenProps.width === 414) {
+    return 'iPhone 6+';
+  } else if (deviceScreenProps.width === 320) {
+    return 'iPhone 5';
+  }
+  return 'iPhone 6';
+};
+
+const deviceScreen = Dimensions.get('window');
```

Comment:
I just kept was what already being used, but yeah I would like to change this in the future